node_modules/gh-auth/tests/test-users.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2014 "Fronteer LTD"
+ * Grasshopper Event Engine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var assert = require('assert');
+var TestsUtil = require('gh-tests');
+var UsersTestsUtil = require('gh-users/tests/util');
+
+var AuthTestsUtil = require('gh-auth/tests/util');
+
+describe('Local user authentication', function() {
+
+    describe('Login', function() {
+
+        /**
+         * Test that verifies that a user can log in
+         */
+        it('verify a user can log in', function(callback) {
+            // Let a global admin create a user
```

Comment:
Yes, but that generates a rest client and user object without providing the password. I think that in this case it's acceptable to create the user manually